BootstrapScriptWithToken/bootstrap.py

```diff
@@ -42,6 +48,14 @@ def getValueFromFile( filename, key ):
             if key in line :
                return line.split( "=" )[ 1 ].rstrip( "\n" )
    return None
+#
+# Set the current time and date from the user input fields
+def setCurrentTimeDate(currentTimeDate, set_timezone):
+   set_cli_privilege = EapiClient(disableAaa=True, privLevel=15)
+   clock_cmds = ['configure', 'clock timezone {}'.format(set_timezone), 'exit', 'clock set {}'.format(currentTimeDate)]
```

Comment:
we are configuring the clock to an exact timestamp, this could cause problems over time. The script is supposed to be hosted on the DHCP server and may not get updated over months.

Reply:
Good point. This is specifically a use case for day 1 onboarding of the switches and the customer does not want to point to an external or public NTP.  NTP synchronization is a good idea when the script is stored in server and untouched for a long time.  Yes the hardcoded date and time would be dated. I'll test NTP as well.